src/sentry/integrations/base.py

```diff
@@ -463,3 +468,72 @@ def notify_remove_external_team(self, external_team: ExternalActor, team: Team)
 
     def remove_notification_settings(self, actor_id: int, provider: str) -> None:
         pass
+
+
+def is_response_success(resp) -> bool:
```

Comment:
nit: we should at least add `Any` as a type hint

Reply:
This looks about the same as the base definition - https://github.com/getsentry/sentry/blob/master/src/sentry/shared_integrations/client/base.py#L122-L126 are you re-defining it since you're redoing the client and it won't inherit from this anymore? I'm a little out of the loop of what you're doing here

Reply:
yes it doesn't inherit from the client anymore and the response class is different